src/cascadia/TerminalSettingsModel/IAppearanceConfig.idl

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "IInheritable.idl.h"
+
+#define INHERITABLE_PROFILE_SETTING(Type, Name) \

```

Comment:
Should this be `INHERITABLE_APPEARANCE_SETTING` so that there's no potential conflict with the `INHERITABLE_PROFILE_SETTING` in `Profile.idl`?

Reply:
Makes sense, changed